doc/howtos/pull-request-guidelines.rst

```diff
@@ -0,0 +1,161 @@
+.. _pull-request-guidelines:
```

Comment:
Thanks for writing this Patrick.
One more place to update to get it into the online documentation set.
Add a line under the .toctree directive in howtos.rst to mention this doc (add pull-request-guidelines to the list)